src/main/java/rs/teslaris/core/controller/ProceedingsPublicationController.java

```diff
@@ -24,13 +29,25 @@ public class ProceedingsPublicationController {
 
     private final ProceedingsPublicationService proceedingsPublicationService;
 
+    private final JwtUtil tokenUtil;
+
+    private final UserService userService;
+
 
     @GetMapping("/{publicationId}")
     public ProceedingsPublicationDTO readProceedingsPublication(
         @PathVariable Integer publicationId) {
         return proceedingsPublicationService.readProceedingsPublicationById(publicationId);
     }
 
+    @GetMapping("/for-event/{eventId}")
```

Comment:
for the JournalPublicationController this endpoint is /my-publications/{journalId}. Can we align this? Maybe we can use api/proceedings-publication/event/{eventId}/my-publications and /api/journal-publication/journal/{journalId}/my-publications. Besides those endpoints for my-publications, we will need also api/proceedings-publication/event/{eventId} and /api/journal-publication/journal/{journalId}, which will return all publications from event or conference (for the event and journal profile page).

Reply:
Ok, I will align the paths as you suggested and add missing methods.